app/views/govuk_publishing_components/components/_checkboxes.html.erb

```diff
@@ -10,6 +10,10 @@
       <%= tag.fieldset class: "govuk-fieldset", "aria-describedby": cb_helper.fieldset_describedby do %>
         <%= cb_helper.heading_markup %>
 
+        <% if cb_helper.description %>
+          <%= tag.p cb_helper.description, id: "#{id}-description", class: "govuk-body" %>
```

Comment:
What's the ID for on this paragraph? Do we need it?

Reply:
I don't think so, I'll remove it